Upgrade Cassy to 1.2.0 and add Azure managed identity (#125)
scalar-labs/scalar-terraform: examples/azure/README.md

### Create Cassandra resources

> To use Cassy to backup Cassandra data (enabled by default), you need to create a storage account before creating resources with `scalar-terraform`.

Reviewer: ```suggestion
Before creating Cassandra resources with `terraform apply`, you probably need to configure for Cassy to manage backups of Cassandra data. 

The first thing you need to do for Cassy is create a storage account in the same resource group as the network resource created in the previous section and create a blob type container in the storage account.

Then, update `example.tfvars` with the container URL as follows.
```

> Note that it must be in the resource group created by the network module in the previous section.

Reviewer: Please delete this if the above suggestion is committed.

> Then, create a blob type container in the storage account, and update `example.tfvars` as follows.

Reviewer: Please delete this if the above suggestion is committed.

```
cassy = {
  storage_base_uri     = "https://yourstorageaccountname.blob.core.windows.net/your-container-name"
  storage_type         = "azure_blob"
}
```

> You can also turn off the use of Cassy by setting its `resource_count` to zero.

Reviewer: ```suggestion
If you don't need Cassy, you can disable it by setting its `resource_count` to zero.
```

```
cassy = {
  resource_count = 0
}
```

For more information on Cassy, please refer to [CassySetup](../../docs/CassySetup.md).

> Then run the terraform command:

Reviewer: ```suggestion
Now it's ready to run the terraform commands:
```
